examples/rustapi_module/tests/test_misc.py

```diff
@@ -4,3 +4,19 @@
 def test_issue_219():
     # Should not deadlock
     rustapi_module.misc.issue_219()
+
+
+def test_args_and_kwargs(benchmark):
+    benchmark(rustapi_module.misc.args_and_kwargs, 1, 2, 3, a=4, foo=10)
+
+
+def test_mixed_args(benchmark):
+    benchmark(rustapi_module.misc.mixed_args, 1, 2, 3, bar=4, foo=10)
+
+
+def mixed_args_py(a, b=2, *args, c=4, **kwargs):
+    (a, b, args, c, kwargs)
```

Comment:
Should there be a `return` here?

Reply:
It'll probably not change the benchmark much, but yes, to exactly match the Rust benchmark we want this here. Thanks for reading 👍